quickjs.c

```diff
@@ -28479,6 +28498,11 @@ JSValue JS_GetModuleNamespace(JSContext *ctx, JSModuleDef *m)
     return js_dup(m->module_ns);
 }
 
+JSValue JS_GetImportAssertion(JSContext *ctx)
+{
+    return ctx->import_assertion;
```

Comment:
```suggestion
    return js_dup(ctx->import_assertion);
```
Or change the return type to JSValueConst but that's much less idiomatic (even internally we only have three functions that do that.)